common/views/components/Body/Body.js

```diff
@@ -1,20 +1,16 @@
 // @flow
 // TODO: Sync up types with the body slices and the components they return
 import {spacing, classNames} from '../../../utils/classnames';
-import {breakpoints} from '../../../utils/breakpoints';
 import AsyncSearchResults from '../SearchResults/AsyncSearchResults';
-import {CaptionedImage, UiImage} from '../Images/Images';
+import {CaptionedImage} from '../Images/Images';
 import Quote from '../Quote/Quote';
 import ImageGallery from '../ImageGallery/ImageGallery';
 import PrismicHtmlBlock from '../PrismicHtmlBlock/PrismicHtmlBlock';
 import FeaturedText from '../FeaturedText/FeaturedText';
 import VideoEmbed from '../VideoEmbed/VideoEmbed';
 import Map from '../Map/Map';
-import WobblyBottom from '../WobblyBottom/WobblyBottom';
-import {PictureFromImages} from '../Picture/Picture';
 import TextLayout from '../TextLayout/TextLayout';
-import ImageLayout from '../ImageLayout/ImageLayout';
-import FullWidthLayout from '../FullWidthLayout/FullWidthLayout';
+import SingleColMax10Layout from '../SingleColMax10Layout/SingleColMax10Layout';
```

Comment:
It's not the hill I'm going to die on, but I think I prefer `Layout10` or `Layout10Col` or `LayoutCol10`. But maybe if we can get a provisional set of layouts from Gareth, just as sketches on a piece of paper, maybe we could spend 20 minutes figuring out a naming convention that we're all happy with?

Reply:
Isn't what makes this Layout stand out is that it is single column?

Reply:
What about `LayoutSingleCol10`?

Reply:
e.g. `Layout1Col10Wide`, `Layout3Col4Wide` (and the fact that that's the _max_ is always tacit).

Reply:
as per, I've moved to `Layout10`.
I'll move the others over in another PR.